velox/exec/tests/HashJoinTest.cpp

```diff
@@ -364,10 +365,10 @@ TEST_F(HashJoinTest, lazyVectors) {
                 .project({"c1 + 1"})
                 .planNode();
 
-  assertQuery(
-      op,
-      {{rightScanId, {rightFile}}, {leftScanId, {leftFile}}},
-      "SELECT t.c1 + 1 FROM t, u WHERE t.c0 = u.c0");
+  AssertQueryBuilder(op, duckDbQueryRunner_)
+      .split(rightScanId, makeHiveSplit(rightFile->path))
+      .split(leftScanId, makeHiveSplit(leftFile->path))
```

Comment:
does it make sense to add another `split` variant that takes a file path (`leftFile->path`) and invokes `makeHiveSplit` in the implementation?

Reply:
probably one for `makeHiveSplits`. The ideal would be to combine both `makeHiveSplits` and `makeHiveSplit` into one.